src/options.c

```diff
@@ -107,8 +107,6 @@ static const struct option lopts[] = {
     {"select",          optional_argument,  NULL,   's'},
     {"thumb",           required_argument,  NULL,   't'},
     {"focused",         no_argument,        NULL,   'u'},
-    /* macquarie dictionary has both spellings */
-    {"focussed",        no_argument,        NULL,   'u'},
```

Comment:
This is unnecessary. You can just do the following inside the zsh completion to exclude it from the completion:

```zsh
    if [[ $lopt == focussed ]]; then
        continue
    fi
```